NICE.Identity.Management/ClientApp/src/views/EditOrganisationUsers/__tests__/__snapshots__/EditOrganisationUsers.test.js.snap

```diff
@@ -90,7 +90,7 @@ exports[`should match the snapshot after data has been loaded 1`] = `
     class="page-header"
   >
     <h1
-      class="page-header__heading"
+      class="page-header__heading "
```

Comment:
what's causing this trailing space?

Reply:
Yeah, I saw this. I'm fairly sure it's coming from the new ds.